Makefile

```
@@ -231,6 +232,15 @@ docker/fv3fit/requirements.txt:
 		external/loaders/setup.py \
 		external/vcm/setup.py
 
+docker/artifacts/requirements.txt:
+	cp constraints.txt $@
```

to allow overwriting the target if it exists without choking.
```suggestion
	cp -f constraints.txt $@
```